spec/govuk_docker/doctor/checkup_spec.rb

```diff
@@ -20,9 +20,9 @@
   context "when the repository is up-to-date" do
     it "should report that it is up-to-date" do
       subject = described_class.new(
-        service_name: service_name,
+        service_name:,
```

Comment:
The [hash value omission feature](https://bugs.ruby-lang.org/issues/14579) seems to have been [added in ruby 3.1.0](https://www.ruby-lang.org/en/news/2021/12/25/ruby-3-1-0-released/), so this commit should probably be `Fix rubocop violations for Ruby >= 3.1`?

Reply:
Oh yes, good catch. That's what I meant 😆